src/reconstruction/refinement/addReactionGEM.m

```diff
@@ -69,6 +69,8 @@
 end
 
 nMet = length(model.mets)+1;
+orignMets = numel(model.mets);

```

Comment:
Do we still need `nMet`, or why could `origMets` not be defined as a function of `nMet`?